go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1 // indirect
+	golang.org/x/net v0.0.0-20201021035429-f5854403a974 // indirect
```

Comment:
plese do not include automatic changes to `go.mod` and `go.sum` since I'll do it anyway while releasing.

Reply:
Thanks, will update.